GOVERNANCE-elections.md

```diff
@@ -0,0 +1,274 @@
+as-needed. The committee currently recognizes this delegated authority for:
+
+- [TODO: List of Delegated Authorities]
+
+<!-- Above put a list of powers of the SC that are being handled by some other
+group or person.  For example, you may have a Marketing Committee that handles
+all work with CNCF marketing, or a different body that handles escalated
+technical disputes.-->
+
+## Committee Meetings
+
+The Steering Committee meets [TODO: frequency of meetings], or as-needed.
+Meetings are held online, and are public by default.
+
+Given the private nature of some of these discussions (e.g. privacy, private
+emails to the committee, code of conduct violations, escalations, disputes
+between members, security reports, etc.) some meetings are held in private.
+<!-- consider setting up a cadence of private vs. public meetings here-->
+
+Meeting notes are available to members of the
+[developer mailing list](TODO).
+Public meetings will be recorded and the recordings made available publicly.
+
+Questions and proposals for changes to governance or community procedures are posted as
+[issues in the community repo](TODO), and
+the Steering Committee invites your feedback there. See
+[Getting in touch](#getting-in-touch) for other options.
+
+## Committee members
+
+Seats on the Steering Committee are held by an individual, not by their
+employer.
+
+The current membership of the committee is currently (listed alphabetically by
+first name):
+
+| &nbsp;                                                         | Member           | Organization |
+| -------------------------------------------------------------- | ---------------- | ------------ |
+
+
+## Decision process
+
+The Steering Committee desires to always reach consensus.
+
+Decisions requiring a vote include: issuing written policy, amending existing
+written policy, creating, removing, or modifying a working group, all spending,
+hiring, and contracting, official responses to publicly raised issues, or any
+other decisions that at least two of the members present decide require a vote.
+
+Decisions are made in meetings when a quorum of more than half of the members
+are present (and all members have been informed of the meeting), and may pass
+with more than half the members of the committee supporting it.
+
+## Getting in touch
+
+There are two ways to raise issues to the steering committee for decision:
+
+1. Emailing the Steering Committee at
+   [steering@project.org](TODO).
+   This is a private discussion list to which all members of the committee have
+   access.
+2. Open an issue on [the community repository](TODO) and indicate that you would like
+   attention from the steering committee.
+
+## Composition
+
+The steering committee has [TODO: Number] seats. These five seats are
+open to any project contributor. See
+[candidate eligibility](#candidate-eligibility) for a definition.
+
+Steering Committee members serve for 2-year terms, staggered in order to
```

Comment:
2-year term ... may be the `2` should be a TODO that should be filled?

Reply:
yah, probably

Reply:
done